docusaurus/docs/get-started/set-up-development-environment.mdx

```diff
@@ -130,3 +141,252 @@ You will also notice that the `docker-compose.yaml` file also has the following
 ```
 
 they are necessary to allow delve to attach to the running process and debug it and should not be used in production environments.
+
+## Set up GitHub workflows
+
+Automate your development process to minimize errors and make it faster and more cost-efficient. The `create-plugin` tool helps you to configure your GitHub actions workflows to help automate your development process.
+
+### The CI workflow
+
+The CI (`ci.yml`) workflow is designed to lint, type check, and build the frontend and backend. It is also used to run tests on your plugin every time you push changes to your repository. The `create-plugin` tool helps to catch any issues early in the development process, before they become bigger problems.
```

Comment:
>The `create-plugin` tool helps to catch any issues early in the development process, before they become bigger problems.

The tool scaffolds things to help catch problems however this sentence gives me the impression it is running in CI.

Reply:
```suggestion
The CI (`ci.yml`) workflow is designed to lint, type check, and build the frontend and backend. It is also used to run tests on your plugin every time you push changes to your repository. Running the CI workflow helps to catch any issues early in the development process, before they become bigger problems.
```